_test/setup.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+trap "exit 1" TERM
+export TOP_PID=$$
+NAMESPACE="${2:-pipelinelib-testing}"
+TRAVIS_REPO_SLUG="${3:-redhat-cop/pipeline-library}"
+TRAVIS_BRANCH="${4:-master}"
```

Comment:
I would suggest removing `TRAVIS_` from these variables in the event we aren't using Travis CI (or for local testing).